data.php

```diff
@@ -188,16 +190,10 @@ function getAllQueries($orderBy) {
             $status = "";
 
             if (substr($tmp, 0, 5) == "query"){
-              $type = substr($exploded[count($exploded)-4], 6, -1);
-              $domain = $exploded[count($exploded)-3];
-              $client = $exploded[count($exploded)-1];
-
-            }
-            elseif ((substr($tmp, 0, 9) == "forwarded" || $exploded[count($exploded)-3] == "pi.hole" || $exploded[count($exploded)-3] == $hostname) && $showpermitted){
-              $status="OK";
-            }
-            elseif ((substr($tmp, strlen($tmp) - 12, 12)  == "gravity.list"  && $exploded[count($exploded)-5] != "read") && $showblocked){
-              $status="Pi-holed";
+                $type = substr($exploded[count($exploded)-4], 6, -1);
+                $domain = $exploded[count($exploded)-3];
+                $client = $exploded[count($exploded)-1];
+                $status = isset($gravity_domains[$domain]) ? "Pi-holed" : "OK";
             }
 
             if ( $status != ""){
```

Comment:
The `if` is obsolete. The code above will always set a `$status` and the two `if` clauses can be combined